pkgs/development/python-modules/embedding-reader/default.nix

```diff
@@ -23,7 +24,8 @@ buildPythonPackage rec {
 
   pythonRelaxDeps = [ "pyarrow" ];
 
-  propagatedBuildInputs = [
+  dependencies = [
+    tqdm
```

Comment:
nit: please keep this list alphabetized

Reply:
oops, sorry about that and thanks for the feedback! I've pushed a change to put the list in alphabetical order.